content/docs/guides/read-replica-data-analysis.md

```diff
@@ -40,7 +40,7 @@ Creating a read replica involves adding a read-only compute endpoint to a branch
 2. Select the branch where your database resides.
 3. Click **Add compute**.
 4. On the **Create Compute Endpoint** dialog, select **Read-only** as the **Compute type**.
-5. Specify the **Compute size** options. You can configure a **Fixed Size** compute with a specific amount of vCPU and RAM (the default) or enable **Autoscaling** and configure a minimum and maximum compute size. You can also configure the **Auto-suspend delay** period, which is the amount of idle time after which your read-only compute is transitioned to an idle state. The default setting is 300 seconds (5 minutes). You can set this value up to 604800 seconds (7 days).
+5. Specify the **Compute size** options. You can configure a **Fixed Size** compute with a specific amount of vCPU and RAM (the default) or enable **Autoscaling** and configure a minimum and maximum compute size. You can also configure the **Suspend compute after a period of inactivity** setting, which is the amount of idle time after which your read-only compute is transitioned to an idle state. The default setting is 5 minutes. You can set this value up to 7 days.
```

Comment:
"idle time" and "idle state" could be thought of as synonymous. Maybe use your other wording: "...after which your compute is automatically suspended" or "..after which your compute is scaled down to zero".

Should we also think about trying to get the billing implications (i.e., the benefit) in there, or is that obvious enough?  Something like: 

"...after which your compute is automatically scaled down to zero (no further billing charges until the compute restarts)."

Thoughts?

Reply:
Thank you. Fixed.